site/content/en/docs/concepts/_index.md

```diff
@@ -57,16 +57,47 @@ network throughput between the Pods.
 ### Quota Reservation
 
 _Quota reservation_ is the process during through which the kueue scheduler locks the resources needed by a workload within the targeted
-[ClusterQueues ResourceGroups](/docs/concepts/cluster_queue/#resource-groups)
+[ClusterQueues ResourceGroups](/site/content/en/docs/concepts/cluster_queue.md#resource-groups)
 
 Quota reservation is sometimes referred to as _workload scheduling_ or _job scheduling_,
 but it should not to be confused with [pod scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
 
 ### Admission
 
-_Admission_ is the process of allowing a Workload to start (Pods to be created). A Workload
-is admitted when it has a Quota Reservation and all its [AdmissionCheckStates](/docs/concepts/admission_check)
-are `Ready`.
+_Admission_ is the process of allowing a Workload to start (Pods to be created). Kueue uses a two-step admission cycle for Workload scheduling: 
+
+- Quota Reservation : When a Workload is submitted, it enters a LocalQueue first. This LocalQueue points to a ClusterQueue which is responsible for managing the available resources. The Kueue scheduler checks if the resources (CPU, memory, GPUs, etc.) requested by the Workload can be satisfied using the targeted ClusterQueue's available quota and resource flavors. If the quota is available, the resources are reserved for this workload and other Workloads are prevented from using the same resources. 
+
+- Admission Checks: After the quota is reserved, Kueue executes all AdmissionChecks configured in the ClusterQueue concurrently. These are pluggable AdmissionChecks controllers that can perform certain validations such as policy checks, compliance, etc.
+These checks can be external or internal and determine if additional criteria are met before the Workload is admitted. The Workload is admitted once all its [AdmissionCheckStates](site/content/en/docs/concepts/admission_check.md) are mark `Ready`.
+
+<h4> Example: Provisioning AdmissionCheck </h4>
+
+Earlier, Kueue admissions were mainly based on quota checks- if sufficient quota existed, the workload is admitted. However, just because the quota (logical resource) is available, doesn't mean there are enough actual(physical) compute resources in the cluster to schedule all pods successfully. This is critical in cloud environments where autoscaling provides nodes on-demand. 
+
+[ProvisioningRequest AdmissionCheck](keps/1136-provisioning-request-support/README.md) handles this issue by ensuring that the physical resources can be provisioned before admitting Workloads.
```

Comment:
Can we fix the link please

Reply:
Yes, it is much better to provide documentation to other doc pages. KEPs we only reference as "supplementary materials", or "see also".